cockroachcloud/authorization.md

```diff
@@ -63,36 +52,86 @@ Org Administrators can manage the organization and its members, clusters, and co
 - [Manage billing for the organization](billing-management.html)
 - [Restore databases and tables from a {{ site.data.products.db }} backup](use-managed-service-backups.html#ways-to-restore-data)
 - [Delete an organization](managing-access.html#delete-an-organization)
+
 {{site.data.alerts.callout_info}}
-In a future release, this role will be deprecated in favor of more fine-grained roles for separately administering organization-level user-management functions, cluster management functions, and billing management functions.
+This role is deprecated in favor of the more fine-grained roles that cover the same permissions:
+
+- [Org Administrator](#org-administrator)
+- [Cluster Administrator](#cluster-administrator)
+- [Billing Coordinator](#billing-coordinator)
 {{site.data.alerts.end}}
   
 ### Org Developer (legacy)
 
 Org Developers can read high-level information for all clusters, and monitor all clusters using DB Console.
 {{site.data.alerts.callout_info}}
-In a future release, this role will be deprecated in favor of more fine-grained roles introduced below.
+This role will is deprecated in favor of more fine-grained roles introduced below.
 {{site.data.alerts.end}}
 
-### Cluster Developer
+### Organization member
+
+This default role is granted to all organization users once they are invited. It grants no permissions to perform cluster or org actions.
+
+### Org Administrator
+
+Users with this role on an organization can:
+
+- [invite users to join that organization](managing-access.html#invite-team-members-to-an-organization)
+- [create service accounts](managing-access.html#create-a-service-account)
+- grant and revoke roles for both [users](managing-access.html#manage-an-organizations-users) and [service accounts](managing-access.html#manage-service-accounts).
+
+This role replaces the now deprecated [Org Administrator (legacy)](#org-administrator-legacy).
 
-Cluster Developers can view the details of clusters and can change their IP allowlist configuration. This role can be granted for specific clusters or for all clusters in the organization.
+### Billing Coordinator
+
+Users with this role in an organization can [manage billing for that organization](billing-management.html) through the {{ site.data.products.db }} console billing page at `https://cockroachlabs.cloud/billing/overview`.
+
+Note that billing can also be managed by the [Org Administrator (legacy) role](#org-administrator-legacy).
+
+### Usage Reader
```

Comment:
You can delete this Usage Reader section. We weren't able to ensure it would be fully read-only, so we've chosen to exclude it for now.